src/Symfony/Component/Translation/Tests/Extractor/PhpAstExtractorTest.php

```diff
@@ -0,0 +1,186 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Translation\Tests\Extractor;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Translation\Extractor\PhpAstExtractor;
+use Symfony\Component\Translation\MessageCatalogue;
+
+class PhpAstExtractorTest extends TestCase
+{
+    /**
+     * @dataProvider resourcesProvider
+     *
+     * @param array|string $resource
```

Comment:
native?

Reply:
`$resource` is passed to `PhpAstExtractor::extract` which accept `iterable|string`. I replace `array|` with `iterable|` here.